.changeset/hot-eyes-run.md

```diff
@@ -0,0 +1,10 @@
+---
+'@segment/analytics-node': minor
```

Comment:
Let's update this to `major`.

Can we also add a note that this only introduces a breaking change for custom HTTP Client users, showing the before and after change?